localplanning.services/src/components/applications/hooks/useResumeApplication.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+import { PUBLIC_PLANX_REST_API_URL } from "astro:env/client";
+import { navigate } from "astro:transitions/client";
+
+type UseResumeApplication = () => {
+  submitEmail: (email: string) => Promise<void>;
+  isLoading: boolean;
+  error: string | null;
+  clearError: () => void;
+}
+
+interface ResumeAPIResponse {
+  message: string;
+}
+
+export const useResumeApplication: UseResumeApplication = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+
+  const submitEmail = async (email: string): Promise<void> => {
+    try {
+      setIsLoading(true);
+      setError(null);
+
+      const response = await fetch(`${PUBLIC_PLANX_REST_API_URL}/resume-application`, {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        // TODO: PlanX API currently requires a team name
+        body: JSON.stringify({ payload: email })
+      });
+
+      if (!response.ok) {
+        throw new Error(`Failed to submit: ${response.status}`);
+      }
```

Comment:
We should probably consider an error monitoring service for LPS, I've added a ticket for this.